tests/runtests.py

```diff
@@ -126,11 +126,7 @@ def setup(verbosity, test_labels, parallel, start_at, start_after):
         test_labels_set.add('.'.join(bits))
 
     if verbosity >= 1:
-        msg = "Testing against Django installed in '%s'" % os.path.dirname(django.__file__)
```

Comment:
Is there a reason behind changing this? It added a nice indicator whether or not the tests were running in parallel.

Non-blocking change though. Other than this, the patch is great.

Reply:
I removed it during iteration because `default_test_processes()` went away, didn't think to restore it. Have done now!